ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3657,4 +3657,22 @@ def test104_edit_order_page_then_overview(self, edit_case):
         else:
             self.assertFalse(self.order_page.confirm_popup(check_only=True))
             self.info('asserting redirection to active table')
-            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
\ No newline at end of file
+            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
+
+    def test105_check_new_fields_are_displayed_in_XSLX(self):
+        """
+        Orders: Export : check that the fields of "Forwarding" , "Report sent by", "validation date" and
+        "validation by" are displaying in excel sheet
+        LIMS-7727
+        """
+        random_order = random.choice(self.orders_api.get_all_orders_json())
+        fields = ['Forwarding', 'Report sent by', 'Validation date', 'Validation by']
```

Comment:
you need to set values to validation to be able to implement step : 5 and 7 in test case jira ticket
you can make use of this 
https://github.com/modeso-open-source/1lims-automation/blob/974dea4d920f014c5f0b8aeaee9e64ccfdc4531e/ui_testing/testcases/extended_tests/test001_order.py#L220 
to line : 224